Cargo.yml

```diff
@@ -118,12 +122,12 @@ dependencies:
   async_std_crate       : { version: ^1.6  , optional: true, package: async-std, features: [ unstable ] }
   tokio                 : { version: ^1    , optional: true }
   pin-utils             : { version: ^0.1  , optional: true }
-
+  crossbeam             : { version: ^0.8  , optional: true }
```

Comment:
Do we still need deps on crossbeam and futures-executor?

Reply:
I needed for GlommioTp. Now I removed it